packages/cli/src/commands/generate/helpers.ts

```diff
@@ -5,9 +5,10 @@ import pluralize from 'pluralize'
 import Listr from 'listr'
 import pascalcase from 'pascalcase'
 import { paramCase } from 'param-case'
-
+import { CommandModule } from 'yargs'
```

Comment:
I like to be explicit when we're only using a type. I don't know how other people feel about this?

```suggestion
import type { CommandModule } from 'yargs'
```

Reply:
Good point! Done.